src/template-analyzer.ts

```diff
@@ -174,7 +180,7 @@ export class TemplateAnalyzer {
 
       if (loc.endOffset < str.length) {
         const fullAttr = str.substring(
-          loc.startOffset + attr.length + 1,
+          loc.startOffset + originalAttr.length + 1,
```

Comment:
ah it was this! because `attr` is the unprefixed attribute, `originalAttr` has the prefix added. good catch

Reply:
Lol tests ftw